src/modules/launcher/Plugins/Community.PowerToys.Run.Plugin.Everything/loc/cs/src/modules/launcher/Plugins/Community.PowerToys.Run.Plugin.VSCodeWorkspaces/Properties/Resources.resx.lcl

```diff
@@ -0,0 +1,77 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
All files added to `src/modules/launcher/Plugins/Community.PowerToys.Run.Plugin.Everything/loc/`
must be deleted.